hologram_dashboard/front-end 2/static/css/style.css

```diff
@@ -0,0 +1,59 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+body {
+    font-size: 10px;
+    min-width: 280px;
+    line-height: normal;
+}
+
+.inline {
+    display: inline-block;
+}
+
+.online {
+    height: 10px;
+    width: 10px;
+    background-color: chartreuse;
+    display: inline-block;
+    border: 1px solid white;
+}
+
+.offline {
+    height: 10px;
+    width: 10px;
+    background-color: red;
+    display: inline-block;
+    border: 1px solid white;
+}
+
+.date {
+    color: blue;
+    display: inline-block;
+}
+
+button{
+    font-size: 10px;
+}
+
+.dashboard {
+    display: "table";
+}
+
+table {
+  font-size: 10px;
+  border-collapse: collapse;
+  border-spacing: 1px;
+}
```

Comment:
nit: add a newline at the end.

Reply:
I added it but it seems to be trimmed out automatically by github